base_layer/mmr/benches/smt.rs

```diff
@@ -19,7 +19,7 @@ fn create_smt() -> SparseMerkleTree<Blake2b<U32>> {
     SparseMerkleTree::<Blake2b<U32>>::new()
 }
 
-pub fn benchmark_sparse_merkle_trees(c: &mut Criterion) {
+pub fn benchmark_smt_insert(c: &mut Criterion) {
```

Comment:
Looking at this more closely, I think there's a subtle error in this benchmark. A fresh tree is created for each sample, but then each iteration of that sample applies the `upsert` operations to the same tree. This means the first iteration will perform inserts, and subsequent iterations will perform updates. The timing data will only be accurate if an insert and update take the same amount of time.